### DISCUSSION TITLE
EnMap import not scaling reflectance

### DISCUSSION THREAD

**Discussion by omkamps** [EnMAP-Box/enmap-box#884] [Q&A]
I tried to import EnMap data via _Project->Add Product_. The importing works fine as long as the folder path is small but the data is not scaled into the 0 to 1 range as mentioned in the information panel. 
I am importing the following metadata file _ENMAP01-____L2A-DT0000035934_20230824T115442Z_001_V010400_20231030T112435Z-METADATA.XML_
Is there a way in EnMap-Box to do the scaling? I tried raster math but then I lost the metadata information such as projection and band names.

**Comment by janzandr** [comment]
This algo should do it:
![image](https://github.com/EnMAP-Box/enmap-box/assets/45030715/a767b2ed-68b4-47ba-8877-e1e8b0cd3b3c)

Unfortunately, the algo is placed in the wrong category :-)... will be fixed.

**Comment by janzandr** [comment]
Also, the scaling bug is already fixed, but not yet released.
